lms/djangoapps/course_api/serializers.py

```diff
@@ -73,10 +75,24 @@ class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     pacing = serializers.CharField()
     mobile_available = serializers.BooleanField()
     hidden = serializers.SerializerMethodField()
+    course_modes = serializers.SerializerMethodField()
```

Comment:
I think we want the catalog service to pull the price from Otto and not LMS. I'm surprised it doesn't already do this.